### ISSUE TITLE
Cannot schedule merges on the day of

### ISSUE THREAD

**Issue by arun-is** [gr2m/merge-schedule-action#13]
If it is already 12am UTC (which is still the day before in pacific time, which I'm on), then I cannot schedule a merge because I get the `is already in the past` error.

**Comment by gr2m** [comment]
I'm not sure what the most elegant way would be to solve this issue. Take the time zone of the author into account? Got any ideas?

**Comment by arun-is** [comment]
I wonder if it is possible to access the `cron` param that is provided in the .yml file. Then when doing the time comparison, it can be a comparing that datetime with now.

**Comment by arun-is** [comment]
For more context, I explicitly set the time I want the action to run using the cron param in the .yml file.

**Comment by gr2m** [comment]
yeah that's what I do, too. I think you should have access, it's part of the source code, so you can just read the file

**Comment by jasongaylord** [comment]
@arun-is So you rolled your own by modifying and are not using the marketplace version, right?

**Comment by arun-is** [comment]
I haven't done anything. If I did, I would have submitted a PR. No time for me in the next month or so unfortunately.

**Comment by jasongaylord** [comment]
@gr2m I'm trying this out with a PR on my site tomorrow. I just set it at 8am.

**Comment by jasongaylord** [comment]
It is working. I posted about it at https://jasong.us/334qCIo. @gr2m

**Comment by gr2m** [comment]
You can pass any string for the schedule that will work with the `new Date()` constructor.

For example

```
/schedule 2020-08-15T19:05:08.139Z
```

**Comment by ruisaraiva19** [comment]
@arun-is @jasongaylord @panduit-bilb we just released v2.4.0 with the support to use the `/schedule` command without needing to set a date.

This lets the action merge pull requests the next time the merge action is scheduled via the cron expressions.

So, when using the `@v2` version of the action, you can take advantage of this feature 😉
